api/HTMLMediaElement.json

```diff
@@ -0,0 +1,2181 @@
+{
+  "api": {
+    "HTMLMediaElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLMediaElement",
+        "support": {
+          "chrome": {
+            "version_added": true
```

Comment:
Please add `"webview_android"` and `"chrome_android"` to all interface members and make the match each other. (Here's an [online JSON](http://jsoneditoronline.org/#/) editor with a copy function that will make this easier.